public/pldoc/js/ui.js

```diff
@@ -0,0 +1,110 @@
+define([
```

Comment:
What's the difference between this file and `` _src/pldoc/static/js/ui.js`` (https://github.com/edx/ux-pattern-library/pull/97/files#diff-1a34c101d67f2fe73266b3f7ecb8f768)?